compiler/damlc/README.md

```diff
@@ -64,6 +64,7 @@ When working on the compiler:
 da-ghcid //compiler/damlc/tests:integration-dev --reload=compiler/damlc/tests/daml-test-files --test=":main --pattern="
 bazel run //compiler/damlc/tests:integration-dev -- --pattern=
 bazel run damlc -- compile $PWD/MyDaml12File.daml
+bazel run damlc -- lint $ $PWD/MyDaml12File.daml
```

Comment:
I’m not particularly fond of adding more things to the README. It mostly gets out of date ime and is only kept up2date by people grepping for things. If you want a list of commands `--help` does the trick.

Reply:
Yep. I'll withdraw this.